config/packages/easy_admin/submit.yaml

```diff
@@ -3,15 +3,21 @@ easy_admin:
         Submit:
             class: App\Entity\Submit
             disabled_actions: ['show']
+            edit:
+                role: SUBMIT_EDIT
+            delete:
+                role: SUBMIT_EDIT
             list:
                 filters: ['talk', 'conference', 'submittedAt', 'status', 'users']
+                role: ROLE_USER
                 fields:
                     - { property: 'talk' }
                     - { property: 'conference' }
                     - { property: 'submittedAt', type: 'date' }
                     - { property: 'status', template: 'easy_admin/Submit/status.html.twig' }
                     - { property: 'users', type: 'array', label: 'Speakers' }
             form:
+                item_permission: SUBMIT_EDIT
```

Comment:
I'm not sure I'm commenting on the right line but in the list, we should filter in the DB, not in PHP

---

I noticed that during you presentation this noon. This is kind of wrong:
We retrieve from the DB many items, but we filter them after.
1. The pagination will be broken
2. We charge the DB / and the PHP application for nothing
=> A SQL filter must be set up to filter the data as soon as possible

But let's do that in another PR